### PR TITLE
Init SyftJS Lib

### PR DESCRIPTION
**Pull request by IonesioJunior**
## Description
This pull request lays the foundation for a powerful Typescript library that handles PySyft requests seamlessly within web applications. PySyft is an open-source framework for privacy-preserving machine learning, and this library will serve as an essential tool to integrate PySyft functionality into web-based projects efficiently.

### Key Features and Enhancements:

- [x]  Basic Typescript lib Scaffolding 
- [x] Auto generate lib Documentation 
- [X] SyftJS Github CI workflow
- [X] SyftJS docker file for testing
- [X] SyftJS Documentation Testing
- [x] Python Primitives implementation/tests
- [x] UID Object implementation/tests 
- [x] SyftVerifyKey implementation/tests

### CODE REVIEW

File: .github/file-filters.yml [hunk 1]
```diff
@@ -73,6 +73,19 @@ frontend:
   - packages/grid/frontend/**/*.ts
   - packages/grid/frontend/**/*.tsx
 
+syftjs:
+  - .github/workflows/pr-tests-syftjs.yml
+  - "**.json"
+  - "**.js"
+  - "**.ts"
+  - "**.yaml"
+  - "**.lock"
```
Comment (tcp) [inline .github/file-filters.yml:82]: We shouldn't listen to all these filetypes. It's enough to hit packages/syftjs for now.

File: .github/workflows/pr-tests-syftjs.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
+name: PR Tests - SyftJS
+
+on:
+  workflow_call:
+
+  pull_request:
+    branches:
+      - dev
+      - main
+      - "0.8"
```
Comment (tcp) [inline .github/workflows/pr-tests-syftjs.yml]:
```suggestion

```

**Comment by madhavajay** [comment]
omg 😱🤯🤤🤩

**Comment by madhavajay** [comment]
@tcp and @IonesioJunior lets discuss how much time we think there is to finish this off, id rather it in the code base than rotting on the PR vine.

**Comment by IonesioJunior** [comment]
I'm closing this PR for now to streamline our open PRs and better prioritize our tasks. We'll retain this branch for future reference.

**Pull request closed without merging**